a.go

```diff
@@ -7,14 +7,14 @@ import (
 	"runtime"
 )
 
-// TF is a type passed to Task's Command function to manage task state.
+// The A object is passed to Task's Action function to manage task state.
 //
-// A Task ends when its Command function returns or calls any of the methods
+// A Task ends when its Action function returns or calls any of the methods
 // FailNow, Fatal, Fatalf, SkipNow, Skip, or Skipf.
 //
 // All methods must be called only from the goroutine running the
-// Command function.
-type TF struct {
+// Action function.
+type A struct {
```

Comment:
Assuming you are taking feedback during work-in-progress:
I understand the need to call the instance variable `a` - yet naming the type also `A` is confusing to me.
Why not call it `TaskState`, or possibly `State` as the documentation describes?

If you refer to `testing.T`, where the inspiration comes from, the package helps here: `testing`. A `goyek.A` is very cryptic.

Reply:
Thanks for the feedback.

[Here](https://github.com/goyek/goyek/issues/62#issuecomment-835706079) you were proposing `goyek.E` (for execution). Maybe that would be better?

I have mixed feeling about `State` as methods like `Log` or `Cmd` look strange for a "state" object.

Reply:
Ah, thank you for keeping good memory.
I see now, in hindsight, that `goyek.E` isn't much better either.
(I might also be influenced by the "clean code" course that I'm about to hold at work this week, where I'll stress the point of expressive naming again)

So, what about `Execution` then?

Reply:
...which, thinking further, sounds a bit coming from law making.
How about `Progress`?

Reply:
I like `Progress` 👍

Reply:
@dertseha Let me know what you think. https://github.com/goyek/goyek/pull/102/commits/6412c04497ac1ecbc760f1e758b712726daf33a6

PS. Do not be afraid if you still do not like it.

Reply:
FYI https://github.com/goyek/goyek/discussions/98#discussioncomment-899790